README.md

```diff
@@ -1,4 +1,26 @@
+
+AlarmClock
+==========
+This gives the option to query whether a specified time period has expired. The utility is made to have a very low CPU clock cycle cost for the client compared to using StopWatch or std::chrono.
+
+The implementation of AlarmClock was made to be  as accurate as possible, with the assumption that low CPU usage and cheap, CPU-wise,  API calls are more important than absolute accuracy in terms of some extra microseconds of expiration overhead. 
```

Comment:
"with the assumption that low CPU usage and cheap, CPU-wise, API calls ..." doesn't sound right. Do you mean, "with the tradeoff of less CPU usage resulting in a less accurate expiration capability"?